src/status_im2/navigation/roots.cljs

```diff
@@ -2,12 +2,11 @@
   (:require [quo2.foundations.colors :as colors]
             [react-native.platform :as platform]
             [status-im2.navigation.view :as views]
-            [status-im2.navigation.state :as state]
-            [status-im2.common.theme.core :as utils.theme]))
+            [status-im2.navigation.state :as state]))
 
 (defn status-bar-options
   []
-  (let [dark-mode? (if (= @state/root-id :shell-stack) (colors/dark?) (utils.theme/dark-mode?))]
+  (let [dark-mode? (if (= @state/root-id :shell-stack) (colors/dark?) true)]
```

Comment:
could you pls elaborate

Reply:
After login we are going to use theme as per user's choice, but before login we only have dark mode.

Reply:
perhaps there's a clearer way to explain this in the code.

Reply:
Yes, will add a comment. Thank you

Reply:
Preferably it can be done with variable/method or property names but yea a comment is ok if that's all that can be done.